src/build/build_step.go

```diff
@@ -1215,57 +1211,16 @@ func (r *progressReader) Read(b []byte) (int, error) {
 	return n, err
 }
 
-// buildMaybeRemotely builds a target, either sending it to a remote worker if needed,
-// or locally if not.
-func buildMaybeRemotely(state *core.BuildState, target *core.BuildTarget, inputHash []byte) (*core.BuildMetadata, error) {
+// build builds a target locally, it errors if a remote worker is needed since this has beeen removed.
+func build(state *core.BuildState, target *core.BuildTarget, inputHash []byte) (*core.BuildMetadata, error) {
 	metadata := new(core.BuildMetadata)
 
-	workerCmd, workerArgs, localCmd, err := core.WorkerCommandAndArgs(state, target)
+	workerCmd, _, localCmd, err := core.WorkerCommandAndArgs(state, target)
```

Comment:
Should we delete the code in `WorkerCommandAndArgs` as well? Looks like it's to do with `$(worker)` which I guess we don't need anymore

Reply:
Yea my aim is to keep this in for now this allows us to give people hints to what is wrong. Without this code in core we aren't able to detect it so easily